plugin/documents.py

```diff
@@ -157,6 +162,7 @@ def on_change() -> None:
         else:
             self.set_uri(view_to_uri(view))
         self._auto_complete_triggered_manually = False
+        self._re_evaluate_if_toggle_diagnostics_panel = False
```

Comment:
How about `_re_evaluate_on_next_diagnostics_update`?

Reply:
`_try_again_on_next_diagnostics_update`?

Reply:
that doesn't really tell what will be evaluated